Package.swift

```diff
@@ -58,11 +58,11 @@ let package = Package(
         "GitHub",
         "Models",
         "Stripe",
-        .product(name: "Either", package: "swift-prelude"),
+        .product(name: "Either", package: "Prelude"),
```

Comment:
There seems to be a lot of weirdness with how SPM package naming works, to the point where we can't even seem to upgrade to SPM tools version 5.2 without possibly renaming all of our upstream packages...